docs/3.0.0-beta.0/cli/CLI.md

```diff
@@ -0,0 +1,264 @@
+# Command Line Interface (CLI)
+
+Strapi comes with a full featured Command Line Interface (CLI) which lets you scaffold and manage your project in seconds.
+
+***
+
+## strapi new
+Create a new project
+
+```bash
+strapi new <name>
+
+options: [--dev|--debug|--quickstart|--dbclient=<dbclient> --dbhost=<dbhost> --dbport=<dbport> --dbname=<dbname> --dbusername=<dbusername> --dbpassword=<dbpassword> --dbssl=<dbssl> --dbauth=<dbauth> --dbforce]
+```
+
+- **strapi new &#60;name&#62;**<br/>
+  Generates a new project called **&#60;name&#62;** and installs the default plugins through the npm registry.
+
+- **strapi new &#60;name&#62; --dev**<br/>
+  Generates a new project called **&#60;name&#62;** and creates symlinks for the `./admin` folder and each plugin inside the `./plugin` folder. It means that the Strapi's development workflow has been set up on the machine earlier.
+
+- **strapi new &#60;name&#62; --debug**<br/>
+  Will display the full error message if one is fired during the database connection.
+
+- **strapi new &#60;name&#62; --quickstart**<br/>
+Use the quickstart system to create your app.
+
+- **strapi new &#60;name&#62; --dbclient=&#60;dbclient&#62; --dbhost=&#60;dbhost&#62; --dbport=&#60;dbport&#62; --dbname=&#60;dbname&#62; --dbusername=&#60;dbusername&#62; --dbpassword=&#60;dbpassword&#62; --dbssl=&#60;dbssl&#62; --dbauth=&#60;dbauth&#62; --dbforce**<br/>
+
+  Generates a new project called **&#60;name&#62;** and skip the interactive database configuration and initilize with these options.
+  - **&#60;dbclient&#62;** can be `mongo`, `postgres`, `mysql`.
+  - **&#60;dbssl&#62;** and **&#60;dbauth&#62;** are available only for `mongo` and are optional.
+  - **--dbforce** Allows you to overwrite content if the provided database is not empty. Only available for `postgres`, `mysql`, and is optional.
+
+See the [CONTRIBUTING guide](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md) for more details.
+
+## strapi start
+Start a Strapi application.
+
+That provide some feature like auto restart when you update a file in you application (except public files).
+
+If you don't use `strapi start` to start you application some plugin that use `strapi.reload();` (content-type-builder plugin for example) can break or not work well.
+
+## strapi generate:api
```

Comment:
If the default template has changed to pull from the database.json make a note, else there needs to be a note that it defaults to mongoose